README.md

```diff
@@ -20,6 +20,7 @@ Each task should contain the following:
 
 - `README.md`: A brief markdown file describing the task to be executed.
 - `Validation.md`: A markdown file describing and justifying the expected state changes for manual validation by multisig signers.
+- `Script.s.sol`: A foundry script that implements post upgrade assertions.
```

Comment:
We'd like to move away from code towards json spec files, so I'd like to use a more specific name to avoid the scope of this file from creeping overtime. 


```suggestion
- `Checks.s.sol`: A foundry script that implements post upgrade assertions.
```

Reply:
What about even more specific with `PostCheck.s.sol` to match the function name being implemented?

Reply:
So the script will both sign and do the post check assertions because it inherits `SignFromJson`

Reply:
We've ended up naming these based on the file it inherits from, either `SignFromJson.s.sol` or `NestedSignFromJson.s.sol`, so let's use that here also